configs/configs.go

```diff
@@ -0,0 +1,12 @@
+package configs
+
+const (
+	DefaultVersion = "development"
+)
+
+//nolint:gochecknoglobals // used to store the version of the built binary
+var Version = DefaultVersion
+
+type Config struct {
+	Version string
+}
```

Comment:
Not sure we need all of this 😆

All you need is a `var`, not a `const`, with the default value being `development`, so:

```suggestion
package configs

var Version = "development"
```

Then when building, if you don't specify the linker flag to overwrite the value, it'll happily stay as `development`. No need for a constant, 2 variables and one struct 😆